sdk/src/ed25519_instruction.rs

```diff
@@ -33,6 +33,14 @@ pub fn new_ed25519_instruction(keypair: &ed25519_dalek::Keypair, message: &[u8])
     let signature = keypair.sign(message).to_bytes();
     let pubkey = keypair.public.to_bytes();
 
+    pairless_ed25519_instruction(pubkey, signature, message)
+}
+
+pub fn pairless_ed25519_instruction(
```

Comment:
nit for naming: since it's just accepting everything and creating the instruction, i was thinking `unchecked` would make that clearest, but I'm open to suggestions
```suggestion
pub fn new_ed25519_instruction_unchecked(
```

Reply:
`unchecked` sounds good to me, I couldn't think of anything better than `pairless` so I was hoping to get another suggestion here